src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Replace the literal '7' with a named constant to explain what this value represents, such as 'LONG_TERM_RENTAL_DAYS'. Do the same for the discount value '50', using a name like 'LONG_TERM_DISCOUNT'.